Restore JSZip for broader compatibility

fregante opened the pull request:
Closes #72

arosell-hsco commented:
@fregante  I will answer here since i can't answer in the issue since it was closed, as for what browser i am using, Chrome 102.0.5005.115 x64 and my OS is W10.

The url i was triying to download was https://download-directory.github.io/?url=https%3A%2F%2Fgithub.com%2FOCA%2Freporting-engine%2Ftree%2F14.0%2Freport_py3o

And the problem still persists even if i open devtools and click "empty cache and hard reload"

This is a current screenshoot of the problem
![image](https://user-images.githubusercontent.com/108001493/179710476-3202a615-4e24-463b-a8df-ed11e14f049f.png)

fregante commented:
Thank you, I think it's currently cached. If a page reload doesn't fix it, run this in the console and refresh the page:

```
navigator.serviceWorker.getRegistrations().then(function(registrations) {
 for(let registration of registrations) {
  registration.unregister()
} })
```

fregante commented:
Fixed now. I had renamed the main branch so the deployment wasn't working anymore.

<img width="701" alt="Screen Shot" src="https://user-images.githubusercontent.com/1402241/179715609-5990b7ed-8c9e-44e9-87ff-7e419fcd7329.png">
